src/redux/categories/categories.js

```diff
@@ -0,0 +1,16 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
This should follow the [duck pattern](https://github.com/erikras/ducks-modular-redux) e.g `const CHECK_STATUS = bookstore/categories/CHECK_STATUS`